README.md

```diff
@@ -1,10 +1,132 @@
-### sneaker-portfolio
-- `docker-compose up`
+# Sneaker Portfolio  
+<img src = readme-media/gifs/portfolio.gif > </img>
 
-### sneaker-server
-- `poetry run uvicorn api.main:app` to run Backend
 
-### sneaker-fronted
-- `npm install`
-- `npm run build` for building static website for FastAPI
-- `npm start`
+  
+
+# [Check out the Website here!](https://nsli.me)
+
+
+- [Sneaker Portfolio](#sneaker-portfolio)
+- [Check out the Website here!](#check-out-the-website-here)
+  - [Features](#features)
+  - [How to use?](#how-to-use)
+    - [Backend](#backend)
+    - [Frontend](#frontend)
+    - [Docker 🐳](#docker-)
+  - [API Endpoints](#api-endpoints)
+  - [Future features](#future-features)
+
+
+## Features 
+
+
+
+
+*[CLICK TO EXPAND]*
+<details>
+    <summary> 360 Degree Animations</summary>
+    <br> 
+    <img src = readme-media/gifs/360.gif > </img>
+</details>
+
+<details>
+    <summary>Drag & Drop </summary>
+    <br> 
+    <img src = readme-media/gifs/draganddrop.gif > </img>
+</details>
+
+<details>
+    <summary>Get Shoe Information & Links</summary>
+    <br> 
+    <img src = readme-media/gifs/information.gif > </img>
+</details>
+
+<details>
+    <summary>Add Shoes</summary>
+    <br> 
+    <img src = readme-media/gifs/add.gif > </img>
+</details>
+
+
+<details>
+    <summary>Remove Shoes</summary>
+    <br> 
+    <img src = readme-media/gifs/remove.gif > </img>
+</details>
+
+<details>
+    <summary>Get Random Shoe 🎲</summary>
+    <br> 
+    <img src = readme-media/gifs/random.gif > </img>
+</details>
+
+<details>
+    <summary>Favorite View ❤️</summary>
+    <br> 
+    <img src = readme-media/gifs/favorite.gif > </img>
+</details>
+
+
+
+
+
+
+## How to use?
+
+The portfolio consists of a FastAPI Backend and a Frontend written in React. For the application to be working, both of the services have to be running. 
+
+**BEFORE YOU START THE APPLICATION:**
+
+- Add a `.env` file in `/sneaker-frontend` similar to the `.env.template` found in the same folder. 
+- The variable `REACT_APP_API_URL` should be configured the URL the application is running on. If you are running it on your local machine, use `http://localhost:8000/` as the value.
```

Comment:
One reads the .env line and goes to the file and adds it, then one comes back and reads this. Need to include this up top or explain it other way around

Reply:
Agreed, I swapped the order and changed explanation accordingly